### PR TITLE
Feature hamburger menu

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Changes Made
- Updated Index.html
- Updated Style.css
- Added Accessibility with app.js 
- Used pair-programming

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const menu = document.querySelector('.menu');
+// const menuItems = document.querySelectorAll(".menuItem");
```
Comment (Mihndim2020) [inline app.js]: - [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

File: index.html [hunk 2]
```diff
@@ -13,23 +13,26 @@
...
-            <li><a href="#form">Contact</a></li>
-            <li>
-              <a href="#form"><img src="img/mail.png" alt="Mail" /></a>
-            </li>
+          <ul class="menu">
+            <li><a class="menuItem" href="#">Portfolio</a></li>
+            <li><a class="menuItem" href="#">About</a></li>
+            <li><a class="menuItem" href="#">Contact</a></li>
+            <li><a class="menuItem " href="#"><img class="menu-img" src="img/mail.png" alt=""></a></li>
+          </ul>
```
Comment (Mihndim2020) [inline index.html]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) for this project, 
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

Please kindly make sure the above requirements are implemented, when a user clicks on an option on your mobile menu, he/she should be taken to that section of the page, and the mobile menu should disappear.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @IsmailChangezi, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 #### I will leave the comments here, since your `Readme file` is not accessible for this `PR` as you did not make any change on it ####

- [OPTIONAL] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. It is a good thing you 
 are using the template from Microverse but please kindly adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your project and delete the sections on which you do not have information for now. This will make your readme file more informative and professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by IsmailChangezi** [comment]
# Added the required Changes.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @IsmailChangezi 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✔️ Good commit messages
✔️ Clean/Organized code

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**